### DISCUSSION TITLE
[Accessibility-Reporting] Schema for automated accessibility reports

### DISCUSSION THREAD

**Discussion by abhina-kumar** [w3c/personalization-semantics#259] [Ideas]
Why well-known urls need to provide schema for automated accessibility reports? Shouldn't it be part of reporting API?

**Comment by lwolberg** [comment]
Discussed on the telecon:
This was raised in relation to Reporting API, and should be discussed in that context. It is not intrinsic to Well Known Destination. 

We do expect that a Well Known Destination could redirect and provide either human-readable or machine-readable results. This is relevant for accessibility statement, report a problem, and other possible destinations.

We are privileging the human readable for now.
